liquibase-integration-tests/src/test/java/liquibase/dbtest/AbstractIntegrationTest.java

```diff
@@ -997,7 +997,16 @@ public void testInvalidSqlThrowsException() throws Exception {
             liquibase.update(new Contexts());
             fail("Did not fail with invalid SQL");
         } catch (CommandExecutionException executionException) {
-            Assert.assertTrue(executionException.getMessage().contains("this sql is not valid and should throw an exception"));
+            boolean dbExceptionFound = false;
+            Throwable cause = executionException.getCause();
+            while (cause != null) {
+                if (cause.getCause() instanceof DatabaseException) {
+                    dbExceptionFound = true;
+                    break;
+                }
+                cause = cause.getCause();
+            }
+            Assert.assertTrue(dbExceptionFound);
```

Comment:
Any reason for this change?

Reply:
Yes, previously we were asserting to check whether an error message was displayed, but then we decided to better validate whether a `DatabaseException` is thrown. To do that we were doing something like: `executionException.getCause().getCause().getCause()`, and to add some more robustness we decided to better do it this way just in case in the future there is any change (any other new exception is thrown) that cause we can't get the specific exception we are looking for as described.

Reply:
You might be able to use the method in `ExceptionUtil` here, which would reduce code duplication.

Reply:
Nice, thanks Steve! I have updated this test by using that `ExceptionUtil` method.